modules/polling/helpers/parseRawTally.ts

```diff
@@ -36,10 +37,31 @@ export function parseRawPollTally(rawTally: RawPollTally, poll: Poll): PollTally
       return valueA.gt(valueB) ? -1 : 1;
     });
 
+  const pluralityResult: PluralityResult[] = Object.keys(poll.options)
+    .map(key => {
+      return {
+        optionId: key,
+        optionName: poll.options[key],
+        mkrSupport: new BigNumber(rawTally.options?.[key]?.mkrSupport || 0),
+        firstPct: rawTally.options?.[key]?.mkrSupport
+          ? new BigNumber(rawTally.options[key].mkrSupport)
+              .div(totalMkrParticipation.toBigNumber())
+              .times(100)
+          : new BigNumber(0),
+        winner: rawTally.options?.[key]?.winner ?? false
+      } as PluralityResult;
+    })
+    .sort((a, b) => {
+      const valueA = a.mkrSupport;
+      const valueB = b.mkrSupport;
+      if (valueA.eq(valueB)) return a.optionName > b.optionName ? 1 : -1;
+      return valueA.gt(valueB) ? -1 : 1;
+    });
+
```

Comment:
This is a fairly ugly way to handle this, but I couldn't figure out how to do it in a single `.map` and make typescript happy.